src/client/mapblock_mesh.cpp

```diff
@@ -661,8 +661,11 @@ static void makeFastFace(const TileSpec &tile, u16 li0, u16 li1, u16 li2, u16 li
 	TODO: Add 3: Both faces drawn with backface culling, remove equivalent
 */
 static u8 face_contents(content_t m1, content_t m2, bool *equivalent,
-	const NodeDefManager *ndef)
+                        const NodeDefManager *ndef, u16 tid1)
```

Comment:
Don't listen to the code checked. The old indents were correct.

Reply:
I agree.